### PR TITLE
Reference documentation for `jobs`

### PR DESCRIPTION
**Pull request by acammack-r7**
Adds documentation for the `jobs` command and a reusable doc fragment for how Metasploit builds ranges of ids and IPs. Minor tweak to the `check` command to use all IPs passed for my own sanity.

Verification
========
- [x] `msfconsole`
- [x] `help jobs` should show extra, helpful help
- [x] In an exploit module `check 127.0.0.1, 127.0.0.2` actually checks those IPs

#11407

### CODE REVIEW

**Comment by busterb** [comment]
LGTM, thanks @acammack-r7

File: documentation/cli/msfconsole.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+### Ranges of IPs
+
+There are several ways to specify ranges of IP addresses that can be mixed
+together. The first way is a list of IPs separated by just a ` ` (ASCII space),
+with an optional `,`. The next way is two complete IP addresses in the form of
+`BEGINNING_ADDRESS-END_ADDRESS` like `127.0.1.44-127.0.2.33`. CIDR
+specifications may also be used, however the whole address must be given to
+Metasploit like `127.0.0.0/8` and not `127/8`, contrary to the RFC.
+Additionally, a netmask can be used in conjunction with a domain name to
+dynamically resolve which block to target. All these methods work for both IPv6
```
Comment (void-in) [inline documentation/cli/msfconsole.md]: A little type. IPv6 should be IPv4 here

Comment (acammack-r7) [reply documentation/cli/msfconsole.md]: Indeed, thanks!

**Comment by busterb** [comment]
## Release Notes

This adds extended documentation for the `jobs` command to msfconsole.

**Comment by h00die** [comment]
it looks like this puts markdown text on the command line, and the `-d` flag used for `info` doesn't open it in a browser.  Am I missing something here?

**Comment by acammack-r7** [comment]
It's a bit a tricky. I definitely didn't want the extra help to be entirely gated behind `help -d` because it would be nearly impossible for newcomers to find (`help help` is currently an Easter egg and none of the readily google-able guides would mention it). Another issue is that any rendered documentation should be able to link between various documents, so the `info -d` trick of rendering just the one page to a tempfile would need to be rethought or augmented. I'm not opposed to adding it for symmetry, but it needs a bit of care to be really useful (my idea would be along the lines of `gem server` when you have docs installed).

**Pull request merged**